src/execution_plan/ops/op_node_by_id_seek.h

```diff
@@ -0,0 +1,52 @@
+/*
+* Copyright 2018-2019 Redis Labs Ltd. and Contributors
+*
+* This file is available under the Redis Labs Source Available License Agreement
+*/
+
+#pragma once
+
+#include "op.h"
+#include "../../parser/ast.h"
+#include "../../graph/graph.h"
+
+#define ID_RANGE_UNBOUND -1
+
+/* Node by ID seek locates an entity by its ID */
+typedef struct {
+    OpBase op;
+    Graph *g;               // Graph object.
+    int nodeRecIdx;         // Position of entity within record.
+    int recLength;          // Size of record.
+    NodeID minId;           // Min ID to fetch.
+    bool minExclusive;      // Include min ID.
+    NodeID maxId;           // Max ID to fetch.
+    bool maxExclusive;      // Include max ID.
```

Comment:
Rename to `minInclusive` and `maxInclusive`

Reply:
renamed.